packages/block-library/src/more/edit.js

```diff
@@ -3,82 +3,66 @@
  */
 import { __ } from '@wordpress/i18n';
 import { PanelBody, ToggleControl } from '@wordpress/components';
-import { Component } from '@wordpress/element';
 import { InspectorControls } from '@wordpress/block-editor';
 import { ENTER } from '@wordpress/keycodes';
 import { getDefaultBlockName, createBlock } from '@wordpress/blocks';
 
-export default class MoreEdit extends Component {
-	constructor() {
-		super( ...arguments );
-		this.onChangeInput = this.onChangeInput.bind( this );
-		this.onKeyDown = this.onKeyDown.bind( this );
+const DEFAULT_TEXT = __( 'Read more' );
 
-		this.state = {
-			defaultText: __( 'Read more' ),
-		};
-	}
+export default function MoreEdit( {
+	// we provide a default value as previously we allowed to save `undefined`, even though it wasn't clear to the UI
```

Comment:
This comment isn't really clear to me. Also, most comments should use sentence capitalization and punctuation.

Reply:
> Also, most comments should use sentence capitalization and punctuation.

I'll keep it in mind. Thanks! -- I changed the comment.